editor/components/post-publish-panel/prepublish.js

```diff
@@ -12,7 +19,20 @@ import PostVisibilityLabel from '../post-visibility/label';
 import PostSchedule from '../post-schedule';
 import PostScheduleLabel from '../post-schedule/label';
 
-function PostPublishPanelPrepublish() {
+function PostPublishPanelPrepublish( {
+	user,
+} ) {
+	const userCanPublishPosts = get( user.data, [ 'post_type_capabilities', 'publish_posts' ], false );
```

Comment:
Can you incorporate #6724 into this pull request?

Reply:
Incorporated #6724 into this PR in [this commit](https://github.com/WordPress/gutenberg/pull/6729/commits/aacf6ca439b019d39b3998ed3f71d1b10821c922).

Thank you so much again for reviewing this @danielbachhuber.